docs/developer/developing.md

```diff
@@ -0,0 +1,49 @@
+# Developing Cincinnati
+
+## Overview
+
+The following sections describe how to build and test the project for local development. 
+
+
+## Prerequisites
+
+* Git
+* Rust
+* [just](https://github.com/casey/just)
+
+*Note:* We recommend using [Rustup](https://github.com/rust-lang/rustup/blob/master/README.md) for Rust toolchain management. 
+
+## Running it locally
+
+Please below commands in different terminal windows 
+```
+$ just run-graph-bulider
+```
+
+```
+$ just run-policy-engine
+```
+
+Here is the command to get graph for "stable-4.2" and for "amd64" architecture
+
+```
+just get-graph-pe "stable-4.2" "amd64"
+```
+
+## Building it locally
+
+TBD
+
+## Running tests locally
+
+To run unit tests
+
+```
+just test
+```
+
```

Comment:
Looks like we also want `just test-net` without the private ones. Would you like to add that recipe and mention it here?

Reply:
sure. My intention is to first get something working merged and then we can improved on this.

Reply:
Also IMO we should be running `./hack/run-all-tests.sh` as part of the Justfile `just test`. we should be consistent with respect to what we run in CI vs on our laptop

Reply:
We could have a recipe named `just run-ci-tests` which does that. I would like to keep a few recipes in there which ease development of specific components.